samples/server/petstore/kotlin-springboot-modelMutable/src/main/kotlin/org/openapitools/model/Pet.kt

```diff
@@ -30,7 +30,7 @@ data class Pet(
     @field:JsonProperty("name", required = true) var name: kotlin.String,
 
     @ApiModelProperty(example = "null", required = true, value = "")
-    @field:JsonProperty("photoUrls", required = true) var photoUrls: kotlin.collections.List<kotlin.String>,
+    @field:JsonProperty("photoUrls", required = true) var photoUrls: kotlin.collections.MutableList<kotlin.String>,
```

Comment:
@rm3l As far as I can tell the mutable models are disabled by default, and since there is not sample project with it active, this change shouldn't happen, or am I wrong?

Reply:
This file got changed right after I launched the `bin/generate-samples.sh` script, as recommended in the PR checklist.
From what I understood after reading the script, it generates samples for [all configs under the `bin/configs` folder](https://github.com/OpenAPITools/openapi-generator/blob/master/bin/generate-samples.sh#L59-L61). It therefore picks the [`kotlin-spring-boot-modelMutable.yaml`](https://github.com/OpenAPITools/openapi-generator/blob/master/bin/configs/kotlin-spring-boot-modelMutable.yaml#L11) file, which generates a sample project with mutable models.
So I guess mutable models are indeed disabled by default, except in this sample project.

Reply:
Thanks for explaining 👍